src/Entity.php

```diff
@@ -1733,58 +1710,41 @@ public static function showNotificationOptions(Entity $entity)
             return false;
         }
 
-        $fn_get_inheritance_label = static function (string $field) use ($ID): string {
-            return self::inheritedValue(
-                self::getUsedConfig($field, $ID, '', ''),
-                false,
-                false
-            );
-        };
-
-        $fn_get_specific_inheritance_label = static function (string $field) use ($entity): string {
-            $tid = self::getUsedConfig($field, $entity->getField('entities_id'));
-            return self::inheritedValue(
-                self::getSpecificValueToDisplay($field, $tid),
-                false,
-                false
-            );
-        };
-
         $inheritance_labels = [
-            'admin_email' => $fn_get_inheritance_label('admin_email'),
-            'admin_email_name' => $fn_get_inheritance_label('admin_email_name'),
-            'from_email' => $fn_get_inheritance_label('from_email'),
-            'from_email_name' => $fn_get_inheritance_label('from_email_name'),
-            'noreply_email' => $fn_get_inheritance_label('noreply_email'),
-            'noreply_email_name' => $fn_get_inheritance_label('noreply_email_name'),
-            'replyto_email' => $fn_get_inheritance_label('replyto_email'),
-            'replyto_email_name' => $fn_get_inheritance_label('replyto_email_name'),
-            'notification_subject_tag' => $fn_get_inheritance_label('notification_subject_tag'),
+            'admin_email' => $entity->getInheritedValueBadge(field: 'admin_email', inherit_parent_value: null),
+            'admin_email_name' => $entity->getInheritedValueBadge(field: 'admin_email_name', inherit_parent_value: null),
+            'from_email' => $entity->getInheritedValueBadge(field: 'from_email', inherit_parent_value: null),
+            'from_email_name' => $entity->getInheritedValueBadge(field: 'from_email_name', inherit_parent_value: null),
+            'noreply_email' => $entity->getInheritedValueBadge(field: 'noreply_email', inherit_parent_value: null),
+            'noreply_email_name' => $entity->getInheritedValueBadge(field: 'noreply_email_name', inherit_parent_value: null),
+            'replyto_email' => $entity->getInheritedValueBadge(field: 'replyto_email', inherit_parent_value: null),
+            'replyto_email_name' => $entity->getInheritedValueBadge(field: 'replyto_email_name', inherit_parent_value: null),
+            'notification_subject_tag' => $entity->getInheritedValueBadge(field: 'notification_subject_tag', inherit_parent_value: null),
```

Comment:
Inherited values should not be shown on root entity.
![image](https://github.com/glpi-project/glpi/assets/33253653/310dd032-4b48-41f6-94f0-5f29751a98fc)

Also, inherited values are invalid on child entities when the root entity value is empty.
![image](https://github.com/glpi-project/glpi/assets/33253653/42475d16-b092-4d8d-88ca-e051beba83f8)